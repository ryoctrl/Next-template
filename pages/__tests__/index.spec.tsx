import * as React from 'react';
import renderer from 'react-test-renderer';
import Home from 'pages/index';
import { shallow, mount } from 'enzyme';
import Head from 'next/head';

describe('Home', () => {
  test('Title is template', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper.find('h1').text()).toEqual('template');
  });

  test(`Home snapshot`, () => {
    const component = renderer.create(<Home />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
