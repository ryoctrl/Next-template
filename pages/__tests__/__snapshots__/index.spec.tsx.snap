// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Home snapshot 1`] = `
<div
  className="container"
>
  <h1>
    fanfare
  </h1>
</div>
`;
